Sources/MarkCodable/MarkCodable.swift

```diff
@@ -47,7 +47,16 @@ final class CodingData {
     }
 }
 
+#if compiler(>=5.7)
```

Comment:
Let's remove the `#if` and the fatal error here? The two code branches are anyways identical to:

```swift
extension Array where Element == Bool {
     mutating func push(_ value: Element) { append(value) }
     mutating func pop() { removeLast() }
 }
```

And that piece of code would work on both 5.6 and 5.7 without warnings. What do you think?

Reply:
sure, I've just removed the redundant one and altered the `Bool` to `Appending` just in case. Thanks Marin.